vignettes/SKG2020_submission/skg2020_submission.Rmd

```diff
@@ -51,14 +52,21 @@ The R package openairegraph, which is available on GitHub as a development versi
 
 The function `openairegraph::oarg_decode()` splits and de-codes each record. Storing the records individually will allow to process the files independent from each other, which is a common approach when working with big data. 
 
-```r
-library(openairegraph)
-library(jsonlite) # tools to work with json files
-library(tidyverse) # tools for tidy data analysis
-# NB, you need to download the file from Zenodo first
-oaire <- jsonlite::stream_in(file("data/h2020_results.gz"))
-openairegraph::oarg_decode(oaire, records_path = "data/records/", 
-  limit = 500, verbose = FALSE)
```

Comment:
I've made this into a chunk which actually is evaluated in the CI to strengthen reproducibiity.